From: Steve Langasek <steve.langasek@ubuntu.com>
Date: Wed, 25 May 2022 18:22:01 +0800
Subject: Don't force inclusion of gnupg in debootstrap

gnupg is now a metapackage depending on large numbers of extraneous
libraries.  It should not be pulled into the base system; we should only
be pulling in the gpgv package instead.
Last-Update: 2018-03-02
---
 functions/defaults.sh | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/functions/defaults.sh b/functions/defaults.sh
index a588df0..5718f07 100755
--- a/functions/defaults.sh
+++ b/functions/defaults.sh
@@ -377,14 +377,6 @@ Set_defaults ()
 		fi
 	fi
 
-	# Include packages on base
-	case "${LB_MODE}" in
-		ubuntu|kubuntu)
-			LB_BOOTSTRAP_INCLUDE="${LB_BOOTSTRAP_INCLUDE:-gnupg}"
-			;;
-
-	esac
-
 	# Exclude packages on base
 	# LB_BOOTSTRAP_EXCLUDE
 
